### DISCUSSION TITLE
scrcpy audio not working

### DISCUSSION THREAD

**Discussion by PrakashK-krd** [community/community#132791] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have android 12 Samsung galaxy.  I downloaded scrcpy 2.5. The video works fine, I can control my phone from my pc. However my audio does not work.

**Comment by knarrff** [comment]
Hi @PrakashK-krd 

You might want to see if others already reported the same problem here: https://github.com/Genymobile/scrcpy/issues?q=is%3Aissue+is%3Aopen++ and if you cannot find your problem mentioned there, open a new issue here: https://github.com/Genymobile/scrcpy/issues/new/choose.
